lib/Models/Catalog/Ows/WebMapServiceCatalogGroup.ts

```diff
@@ -276,6 +277,21 @@ class GetCapabilitiesStratum extends LoadableStratum(
       this.catalogGroup.hideLegendInWorkbench
     );
 
+    // Copy over ExportWebCoverageTraits if `linkedWcsUrl` has been set
+    // See WebMapServiceCatalogGroupTraits.perLayerLinkedWcs for more info
+    if (this.catalogGroup.perLayerLinkedWcs?.linkedWcsUrl) {
+      updateModelFromJson(model, stratum, {
+        // Copy over all perLayerLinkedWcs objects
+        ...this.catalogGroup.traits.perLayerLinkedWcs.toJson(
+          this.catalogGroup.perLayerLinkedWcs
+        ),
+        // Override linkedWcsCoverage with layer.Name (or Title if Name is undefined)
+        linkedWcsCoverage: layer.Name ?? layer.Title
```

Comment:
wait why is it sometimes layer.Name and othertimes layer.name for different catalog types?
Also, it may be worth creating a helper function `getName: (layer: CapabilitiesLayer) => string` that does the resolution as there are other calls to just layer.Name and i'm not sure if they need to fall back to layer.Title. E.g lib/Models/Catalog/Ows/WebMapServiceCatalogGroup.ts:260

Reply:
Sorry I should have just used `layer.Name` here. I have clarified the difference between the two [here](https://github.com/TerriaJS/terriajs//blob/4453b2a553699853614c156f9137e7038f6bf083/lib/Models/Catalog/Ows/WebMapServiceCapabilities.ts#L55-L69)

```ts
export interface CapabilitiesLayer {
  readonly _parent?: CapabilitiesLayer;
  /** ### Adapted from WMS 1.3.0 spec:
   *
   * #### 7.2.4.2
   *
   * A number of elements have both a `<Name>` and a `<Title>`.
   * The `Name` is a text string used for machine-to-machine communication while the `Title` is for the benefit of humans.
   *
   * #### 7.2.4.6.3
   *
   * If, and only if, a layer has a `<Name>`, then it is a map layer that can be requested by using that `Name` in the `LAYERS` parameter of a GetMap request.
   * If the layer has a `Title` but no `Name`, then that layer is only a category title for all the layers nested within.
   * A client shall not attempt to request a layer that has a `Title` but no `Name`.
   */
  readonly Name?: string;
  readonly Title: string;
```